src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +57,79 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        // Register a new user
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+
+        String email = "test@example.com";
+        String password = "securePassword";
+
+        try {
+            User registeredUser = authenticationService.register(email, password);
+            System.out.println("User registered: " + registeredUser);
+        } catch (RegistrationException e) {
+            System.out.println("Registration failed: " + e.getMessage());
+        }
+
+        // Login with correct email and password (retrieve the registered user)
+        User user = null;
+        try {
+            user = authenticationService.login(email, password);
+            System.out.println("User logged in: " + user);
+        } catch (AuthenticationException e) {
+            System.out.println("Authentication failed: " + e.getMessage());
+        }
+
+        if (user != null) {
+            // Add sessions to shopping cart
+            ShoppingCartService shoppingCartService =
+                    (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+            shoppingCartService.addSession(tomorrowMovieSession, user);
+            shoppingCartService.addSession(yesterdayMovieSession, user);
+            System.out.println("Shopping cart before completing order:");
+            System.out.println(shoppingCartService.getByUser(user));
+
+            // OrderService Tests
+            OrderService orderService =
+                    (OrderService) injector.getInstance(OrderService.class);
+
+            System.out.println("\n=== Test 1: Create an order with an empty cart ===");
+            ShoppingCart emptyCart = shoppingCartService.getByUser(user);
+            Order emptyOrder = orderService.completeOrder(emptyCart);
```

Comment:
Creating an order with an empty cart (`emptyCart`) might not be meaningful. Typically, an order should contain at least one item. Consider adding a check to prevent orders from being created with empty carts.